src/func_definition.jl

```diff
@@ -82,6 +72,10 @@ for (def, ns) in deftree
       nns = unique(collect(v[1] for v in nsp))
       # (length(nsp) > 1) && warn("which name ? $(join(nns, "," ))")
       sfn        = Symbol(nsp[1][1])
+      if sfn==:*
+          # TODO Sort out what to do here.
+          sfn = :unclear_what_to_do
+      end
```

Comment:
@fredo-dedup On ``master`` what happens is that there is a function generated with the name ``vl*``. That to me seems more like an error? Or not? I've special cased it here for now, but there is probably a better, proper fix.